libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateAlliedProjectile.cpp

```diff
@@ -26,9 +26,12 @@ namespace ecs
             .addComponent<Life>(1)
             .addComponent<Damage>(damage)
             .addComponent<DamageRadius>(5)
-            .addComponent<Collidable>()
-            .addComponent<AlliedProjectile>(ally.getComponent<Networkable>().id);
-
+            .addComponent<Collidable>();
+        if (ally.contains<Networkable>()) {
+            entity.addComponent<AlliedProjectile>(ally.getComponent<Networkable>().id);
+        } else {
+            entity.addComponent<AlliedProjectile>(ally.getComponent<Player>().playerIdentifier);
```

Comment:
Why did you add that ?

Reply:
To add the AlliedProjectileComponent with id of the player in solo mod